github_test.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"testing"
+	"os"
+)
+
+func TestGetListOfReleasesFromGitHubRepo(t *testing.T) {
+	cases := []struct {
+		repoUrl          string
+		lastReleaseTag   string
+		firstReleaseTag  string
+		gitHubOAuthToken string
+	}{
+		// These tests are obviously brittle. Suggestions welcome on a more durable way to test.
```

Comment:
The brittleness of the tests was because I was using public repos whose releases may change. But I solved that problem by creating dummy repos solely as test fixtures. Adjusted comment accordingly.